lib/drip/client/shopper_activity.rb

```diff
@@ -37,6 +37,27 @@ def create_order_activity_event(data = {})
         make_json_request :post, "v3/#{account_id}/shopper_activity/order", data
       end
 
+      # Public: Create a batch of order activity events.
+      #
+      # options    - Required. An array of hashes containing additional order options.
+      #                       Refer to the Drip API docs for the required schema.
+      #
+      # Returns a Drip::Response.
+      # See https://developer.drip.com/#create-or-update-a-batch-of-orders
+      def create_order_activity_events(records = [])
+        records.each_with_index do |rec, i|
```

Comment:
I don't like such abbreviations (`rec`). I'd prefer either `order` or `record`.

Reply:
I probably prefer the short version, but I also don't care much. Change made.